declarations/Amazon Mechanical Turk.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Amazon Mechanical Turk",
+  "documents": {
+    "Privacy Policy": {
+      "fetch": "https://www.amazon.com/gp/help/customer/display.html/ref=footer_privacy?ie=UTF8&nodeId=468496",
```

This URL is equivalent but simpler (and thus hopefully more stable).
```suggestion
      "fetch": "https://www.amazon.com/gp/help/customer/display.html?nodeId=468496",
```